RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitECAL_cfi.py

```diff
@@ -20,7 +20,8 @@
              qualityTests = cms.VPSet(
```

Comment:
with applySelectionsToAllCrystals = True, I think that we can go back to setting of srFlags to be non-empty.

Reply:
With the flag applySelectionsToAllCrystals set to true, setting the srFlags will not have any effect.
Maybe we gain time leaving it false, so that the collection is not even retrieved.